etc/sh/profile.d/99-private.sh

```diff
@@ -0,0 +1,11 @@
+# Load private profile.d snippets
```

Comment:
This used to be already done with `etc/sh/profile` (now moved into `etc/sh/profile.d/90-private.sh` in cea0a059e4f05ee47c53628ed8e8e219c605923b). Was that somehow missed?

Anyway, this seems unnecessary to me.

Reply:
Somehow missed that completely…. I've removed the commit.